tasks/Task.cpp

```diff
@@ -27,17 +29,58 @@ bool Task::configureHook()
     m_no_actuation_pwm_command = _no_actuation_pwm_command.get();
     m_cmd_in_mode = _cmd_in_mode.get();
     m_helices_alignment = _helices_alignment.get();
-    m_cmd_to_pwm_lut = std::make_unique<PWMTable>(
-        loadPWMTable(_command_to_pwm_table_file_path.get())
-    );
+    m_cmd_to_pwm_lut =
+        std::make_unique<PWMTable>(loadPWMTable(_command_to_pwm_table_file_path.get()));
+
+    auto size = m_helices_alignment.size();
+    m_ramp_states.resize(size);
+    m_ramp_rates = _cmd_ramp.get();
+    m_ramp_initial_values = _ramp_initialization_values.get();
```

Comment:
I really don't think it makes sense to have this as a property. The initial value *must* be the no_actuation value as it is the only value the ESCs will actually accept.